gallery/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicatorPage.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxVulnerabilityIndicatorExample from './NxVulnerabilityIndicatorExample';
+import NxVulnerabilityIndicatorColumnExample from './NxVulnerabilityIndicatorColumnExample';
+import NxVulnerabilityIndicatorGridExample from './NxVulnerabilityIndicatorGridExample';
+
+const nxVulnerabilityIndicatorCode = require('./NxVulnerabilityIndicatorExample?raw'),
+    nxVulnerabilityIndicatorColumnCode = require('./NxVulnerabilityIndicatorColumnExample?raw'),
+    nxVulnerabilityIndicatorGridCode = require('./NxVulnerabilityIndicatorGridExample?raw');
+
+const NxVulnerabilityIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Basic style for small indicator which displays a number representing a count of violations, and a short pre-set
+        text string which displays the vulnerability severity.
+      </p>
+      <p className="nx-p">
+        Three basic layouts have been demonstrated below. It is expected that one of these should satisfy any
+        requirment.
+      </p>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
```

Comment:
As a component, we need to document the props, and probably do not want to document the internal classes.

Reply:
> As a component, we need to document the props, and probably do not want to document the internal classes.

Right, done here: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/ba74de4ac6db5e7c7249a4abf7f83f110cb4777f